app/src/features/surveys/animals/profile/captures/components/capture-card-details/components/ReleaseDetails.tsx

```diff
@@ -18,7 +18,7 @@ interface IReleaseDetailsProps {
 export const ReleaseDetails = (props: IReleaseDetailsProps) => {
   const { capture } = props;
 
-  const releaseTimestamp = capture.release_timestamp;
+  const releaseTimestamp = capture.release_date;
```

Comment:
nit: rename `releaseTimestamp` to `releaseDate` (just for consistency).

Reply:
fixed